pkg/api/registered/registered.go

```diff
@@ -72,6 +74,11 @@ func init() {
 	RegisteredGroupVersions = supportedVersions
 }
 
+// ValidateGroupVersion declares a groupVersion as valid.
+func ValidateGroupVersion(groupVersion unversioned.GroupVersion) {
```

Comment:
a name more like `AddValidGroupVersion` Would be clearer, but why are we adding another place to register groupversions?